build.js

```diff
@@ -116,9 +116,13 @@ function renderTableHeaderRow(writer, maximumLevel) {
     // SDK columns
     // eslint-disable-next-line no-restricted-syntax
     for (const sdkManifestSuffix of sdkManifests.keys()) {
-      rowWriter.class(`px-1 text-center ${commonCellStyle}`);
+      rowWriter.class(`px-1 align-middle ${commonCellStyle}`);
       rowWriter.cell((cellContentWriter) => {
+        cellContentWriter.write(
```

Comment:
I would collapse this statement onto a single line. You're not really approaching any line length limits and I think having it laid out like this suggests more impact than it has, if that makes sense. On one line would make the code more readable on a quick scan.